### PR TITLE
Compares with files example

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Description
Added the example that shows how to compare a response payload to a file

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/use-cases/use-drive.md [hunk 1]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+  mode: structure
+  strict: "true"
+- id: assert-compares
+  expression1: sample_payload
+  expression2: payload
+  mode: values
+  strict: "true"
+```
+
+:::Note Text Mode
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/use-drive.md:478]:
```suggestion
:::note Text Mode
```

File: docs/api-testing/use-cases/use-drive.md [hunk 2]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
       mode: json
   expression: myUserDetails
 ```
+
+## Comparing the Response to a File
+
+The last example considered is when you need to compare the response from a call to a "template" to verify that it is precisely the same. Also, in this case, you must save the template in the Vault Drive and compare it with the response.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/use-drive.md:366]:
```suggestion
For this last example consider a case where you need to compare the response from a call to a "template" to verify that they are the same.  You can  save the template in the Vault Drive and compare it with the response.
```

File: docs/api-testing/use-cases/use-drive.md [hunk 3]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+
+### Step 1: Uploading the File in Vault Drive
+
+The first thing is uploading the file into the Vault. You can follow the [same steps as in the first example](#step-1-uploading-the-file-in-vault-drive), but this time you will use the following payload, saved as _user.json_:
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/use-drive.md:370]:
```suggestion
First, upload file in the Vault. You can follow the [same steps as in the first example](#step-1-uploading-the-file-in-vault-drive), but this time you will use the following payload, saved as _user.json_:
```

File: docs/api-testing/use-cases/use-drive.md [hunk 4]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+Next, you can compare the file to the response using the **Assert compares** component.
+
+1. Add the **Assert Compares** component.
+
+   - Expression 1 - for example `sample_payload`
+   - Expression 2 - for example `payload`
+   - Mode - for example `structure`
+   - Strict - for example `No`
+
+   In **Expression 1** and **Expression 2**, you have to enter the variable names you have entered for the file and the call.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/use-drive.md:436]:
```suggestion
   In **Expression 1** and **Expression 2**, enter the variable names you have entered for the file and the call.
```

File: docs/api-testing/use-cases/use-drive.md [hunk 5]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+   Mode `structure` means you will compare the structure of the two json.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/use-drive.md:437]:
```suggestion
   Mode `structure` means you will compare the JSON structure of the template and the response
```

File: docs/api-testing/use-cases/use-drive.md [hunk 6]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+  strict: "true"
+- id: assert-compares
+  expression1: sample_payload
+  expression2: payload
+  mode: values
+  strict: "true"
+```
+
+:::Note Text Mode
+When you need to compare a string such as an Authorization token, or an ID (not necessarily from a file) the correct **Mode** to set up is `text`.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/use-drive.md:479]:
```suggestion
When you need to compare a string such as an Authorization token, or an ID (not necessarily from a file) you 1text1 **Mode**.
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/api-testing/use-cases/use-drive.md [hunk 7]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+
+1. Add the **Assert Compares** component.
+
+   - Expression 1 - for example `sample_payload`
+   - Expression 2 - for example `payload`
+   - Mode - for example `structure`
+   - Strict - for example `No`
+
+   In **Expression 1** and **Expression 2**, enter the variable names you have entered for the file and the call.
+   Mode `structure` means you will compare the JSON structure of the template and the response
```
Comment (LoryApiFortress) [inline docs/api-testing/use-cases/use-drive.md:437]:
```suggestion
   Mode `structure` means you will compare the JSON structure of the template and the response.
```

File: docs/api-testing/use-cases/use-drive.md [hunk 8]
```diff
@@ -361,3 +360,121 @@ The following example saves the values in variables:
...
+  strict: "true"
+- id: assert-compares
+  expression1: sample_payload
+  expression2: payload
+  mode: values
+  strict: "true"
+```
+
+:::note Text Mode
+When you need to compare a string such as an Authorization token, or an ID (not necessarily from a file) you 1text1 **Mode**.
```
Comment (LoryApiFortress) [inline docs/api-testing/use-cases/use-drive.md:479]:
```suggestion
When you need to compare a string such as an Authorization token, or an ID (not necessarily from a file) you can choose `text` **Mode**.
```

**Comment by LoryApiFortress** [comment]
@lysannep I tried a lot of times, but there's no way to get the test passes. Can you please take a look? Thanks

**Pull request merged**